admin-dev/filemanager/dialog.php

```diff
@@ -347,7 +347,7 @@
     ?>"/>
 	<input type="hidden" id="descending" value="<?php echo $descending ? "true" : "false";
     ?>"/>
-	<?php $protocol = 'http';
+	<?php $protocol = (!empty($_SERVER['HTTPS']) && $_SERVER['HTTPS'] !== 'off' || $_SERVER['SERVER_PORT'] == 443) ? "https" : "http";
```

Comment:
You can use `Tools::getShopProtocol()` instead :)

Reply:
I didn't know about that function... 
You are right that will be a better solution :)
Should I create a new pull request?

Reply:
No, you can edit this one :)

Reply:
Sorry for the delay :)
I have change it to use the `Tools::getShopProtocol()`